icons/folder_type_public.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_public</title><path d="M27.5,5.5H18.2L16.1,9.7H4.4V26.5H29.6V5.5Zm0,4.2H19.3l1.1-2.1h7.1Z" style="fill:#00a23c"/><path d="M22.573,16.294a3.816,3.816,0,0,1,1.776,2.835,3.079,3.079,0,1,0-1.776-2.835ZM20.555,22.6a3.08,3.08,0,1,0-3.08-3.08A3.08,3.08,0,0,0,20.555,22.6Zm1.306.21H19.248A3.948,3.948,0,0,0,15.3,26.753v3.2l.008.05.22.069a17.95,17.95,0,0,0,5.362.865,11.007,11.007,0,0,0,4.682-.879l.206-.1H25.8v-3.2A3.947,3.947,0,0,0,21.861,22.81Zm5.1-3.179H24.364a3.8,3.8,0,0,1-1.171,2.644,4.686,4.686,0,0,1,3.347,4.484v.985a10.612,10.612,0,0,0,4.133-.868l.206-.1H30.9v-3.2A3.948,3.948,0,0,0,26.957,19.63Zm-11.806-.21a3.059,3.059,0,0,0,1.638-.475,3.812,3.812,0,0,1,1.433-2.431c0-.058.009-.115.009-.173a3.08,3.08,0,1,0-3.079,3.079Zm2.766,2.853a3.8,3.8,0,0,1-1.171-2.629c-.1-.007-.191-.015-.289-.015H13.843A3.948,3.948,0,0,0,9.9,23.573v3.2l.008.049.22.069a18.489,18.489,0,0,0,4.441.835v-.965A4.687,4.687,0,0,1,17.917,22.274Z"/></svg>
```

Comment:
I'd suggest to change the color of the user icon to #AAFFA2

![image](https://cloud.githubusercontent.com/assets/696981/26755893/3248b99e-4897-11e7-9943-ed66b4cc0174.png)